skycultures/greek_leidenAratea/description.en.utf8

```diff
@@ -0,0 +1,325 @@
+	</tr>
+	<tr valign="top">
+		<td> Triangulum, Tri </td>
+		<td> &Tau;&rho;&#943;&gamma;&omega;&nu;&omicron;&sigmaf; </td>
+		<td> Triangle </td>
+		<td> <img width="355" src="descr_pics/aratea_small_tri.jpg"/>	</td>
+	</tr>
+	<tr valign="top">
+		<td> Aries Ari </td>
+		<td> &Kappa;&rho;&iota;&#972;&sigmaf; </td>
+		<td> Ram <br> (jumping through a ring which is a symbol for the year; Roman new year was in spring) </td>
+		<td> <img width="255" src="descr_pics/aratea_small_ari.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Taurus, Tau </td>
+		<td> &Tau;&alpha;&upsilon;&#771;&rho;&omicron;&sigmaf; </td>
+		<td> Bull 	</td>
+		<td> <img width="255" src="descr_pics/aratea_small_tau.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> M45, Pleiades </td>
+		<td>   </td>
+		<td> The Pleiades <br>or Seven Sisters	</td>
+		<td> <img width="255" src="descr_pics/aratea_small_plei.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Gemini, Gem </td>
+		<td> &Delta;&#943;&delta;&upsilon;&mu;&omicron;&iota; </td>
+		<td> Twins </td>
+		<td> <img width="255" src="descr_pics/aratea_small_gem.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Cancer Cnc 		</td>
+		<td> &Kappa;&alpha;&rho;&kappa;&#943;&nu;&omicron;&sigmaf; </td>
+		<td> Cancer <br><i>The cancer was lightened. </i> </td>
+		<td> <img width="255" src="descr_pics/aratea_small_cnc.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Leo, Leo<</td>
+		<td> &Lambda;&#941;&omega;&nu; </td>
+		<td> Lion <br> mirrored depiction</td>
+		<td> <img width="255" src="descr_pics/aratea_small_leo.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Virgo, Vir 		</td>
+		<td> &Pi;&alpha;&rho;&theta;&#941;&nu;&omicron;&sigmaf; </td>
+		<td> Young Lady <br> Apparently, the manuscript only contains one drawing for figures that are depicted twice: like Sagittarius is depicted as centaur and the constellation centaur is missing, 
+Andromeda is painted as young lady but the second young lady for the constellation of Virgo is missing. </td>
+		<td> missing		</td>
+	</tr>
+	<tr valign="top">
+		<td> Libra, Lib </td>
+		<td> &Chi;&eta;&lambda;&alpha;&#943; </td>
+		<td> Balance (Latin, since 1st century CE in Geminos and Manilius; transferred in a style epoch of Babylonian revival),
+		<br> Claws [of the Scorpion] (Greek) <br> 
+			The claws of the Scorpion indeed to cover this area.</td>
+		<td> missing	</td>
+	</tr>
+	<tr valign="top">
+		<td> Ophiuchus et Serpens, Scorpius, <br> Oph+Ser and Sco   </td>
+		<td> &#702;&Omicron;&phi;&iota;&omicron;&#973;&chi;&omicron;&sigmaf; , &#702;&#714;&Omicron;&phi;&iota;&sigmaf; and
+		&Sigma;&kappa;&omicron;&rho;&pi;&#943;&omicron;&sigmaf; </td>
+		<td> Snake Carrier and Scorpion (both in the zodiac ever since)<br> mirrored depiction </td>
+		<td> <img width="255" src="descr_pics/aratea_small_oph.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td> Sagittarius, Sgr 	</td>
+		<td> &Tau;&omicron;&xi;&#972;&tau;&eta;&sigmaf; 	</td>
+		<td> Bow Shooter </td>
```

Comment:
"Archer" would be better, or is there a particular reason for this wording?

Reply:
sometimes, there are Dutch or French vocab in my head ... but honestly I don't see a problem